nengo/spa/assoc_mem.py

```diff
@@ -60,8 +54,9 @@ def __init__(self, input_vocab, output_vocab=None,  # noqa: C901
                  default_output_key=None, threshold=0.3,
                  inhibitable=False, wta_output=False,
                  wta_inhibit_scale=3.0, wta_synapse=0.005,
```

Comment:
Oh, I didn't mean remove them from the constructor. I just meant use parameters for them internally, so that a user could set the e.g. wta_synapse for all associative memories with one line. Or is that not possible with our current config system?